hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/Util.java

```diff
@@ -44,5 +44,68 @@ public static String decodeBase64(String stringToDecode) throws UnsupportedEncod
 
         return decodedString;
     }
+	
+	/**
+	 * Takes a data buffer and scramble the contents using a simple algorithm
+	 * 
+	 * @param aByte The buffer the contents of which are to be scrambled. May
+	 */
+
+	public static void scrambleData(byte[] aByte, byte decodeSeed) {
```

Comment:
If we're adding/moving these methods to a new class (like Util), can we just have the HandshakeServer use them too?
From what I can tell these are just being used by the unit tests right now.